src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -418,37 +420,61 @@ private void testDeleteInstructorAction() {
         
         ______TS("delete instructor then cancel");
         
-        courseEditPage.clickDeleteInstructorLinkAndCancel();
+        courseEditPage.clickDeleteInstructorLinkAndCancel(1);
         assertNotNull(BackDoor.getInstructorAsJsonByGoogleId(instructorId, courseId));
         
         ______TS("delete instructor successfully");
         
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
         String expectedMsg = "The instructor has been deleted from the course.";
         courseEditPage.verifyStatus(expectedMsg);
         
-        ______TS("delete all instructors");
+        ______TS("delete all other instructors");
         
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
-        courseEditPage.clickDeleteInstructorLinkAndConfirm();
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(1);
+        courseEditPage.clickDeleteInstructorLinkAndConfirm(2);
         
-        ______TS("deleted own instructor role and redirect to courses page");
+        ______TS("test the only registered instructor with the privilege to modify instructors cannot be deleted");
         
-        // Change login id to another instructor
+        // Create an registered instructor with all privileges except modifying instructors
+        InstructorPrivileges privilege = new InstructorPrivileges(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER);
+        privilege.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR, false);
+        InstructorAttributes instructor = new InstructorAttributes("InsCrsEdit.reg", courseId, "Teammates Reg", "InsCrsEdit.reg@gmail.tmt", 
+                                        Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM, "Teammates Reg", privilege);
+        BackDoor.createInstructor(instructor);
```

Comment:
A few lines in the block above can be shortened, to avoid horizontal scrolling in diff view.